src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -115,7 +118,7 @@ public function getItemFromIri(string $iri, array $context = [])
      */
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
-        $resourceClass = $this->getObjectClass($item);
+        $resourceClass = null !== $this->resourceClassResolver ? $this->resourceClassResolver->getResourceClass($item) : $this->getObjectClass($item);
```

Comment:
```suggestion
        $resourceClass = null === $this->resourceClassResolver ? $this->getObjectClass($item) : $this->resourceClassResolver->getResourceClass($item);
```

Reply:
Same as above. I don't see what's the benefit of reserving this. It makes the code harder to follow.